packages/utils/src/getFeatureFlag.ts

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Analytics } from '@deriv-com/analytics';
+
+const waitForGrowthbook = () => {
+    return new Promise(resolve => {
+        const checkAnalytics = () => {
+            if (typeof Analytics !== 'undefined' && Analytics.getInstances()?.ab !== undefined) {
+                resolve(Analytics);
+            } else {
+                setTimeout(checkAnalytics, 100); // Retry every 100ms
```

Comment:
given "polling" is used, I'm guessing there is no direct way to check? e.g. promise from analytics or something, 

semi-poling every 100ms will add on average 50ms extra to resolution time (e.g. sometimes it will resolve 10ms after the analtics are actually ready, sometimes 90ms after analytics are ready)

Reply:
I have lowered it to 50ms. We can't go to promise approach because I needed to wait for Analytics to get initialized from other part of the code, means that I still need to add timeout after I get the availability of Analytics.